java/daikon/chicory/Runtime.java

```diff
@@ -168,6 +173,7 @@ public TerminationMessage() {
   // the trace file, don't record it.
   private static boolean invokingPure = false;
 
+  /*@Holding("Runtime.class")*/
```

Comment:
Why is the annotation on dontProcessPpts but not on startPure or endPure?  Those methods use the same field.